microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,97 @@
+
+In general, the website's internationalization user experiences are only as good as the regional code that they're able to determine for the current user. 
+
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+
+To set the user's language preferences, Microsoft Edge gets the default values from the OS when Microsoft Edge is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.
+
+| Language or region data | User impact |
+|--------|-------------------------|
+| **Preferred language** in the **setting** page | The language selected for rendering Microsoft Edge. This language/region is also used for formatting values such as date, time, and numbers. |
+|  `accept-language` HTTP header | A list of regional codes sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
+| `navigator.language` API (and similar) | Allow a Website to display the right language and regional formatting for user input and other content produced locally. |
+
+## Regional Data Display options
+Users have three options on Microsoft Edge to select the level of information their browser will share about their regions:
+
+* Limited (default).
+* Always (not recommended).
+* Never 
+
+#### Limited  (default)
+The default, recommended value is **Limited**. This setting uses only valid combinations of the user's current browser display language preference and the given region value. **Limited** ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in mismatched combinations). The **Limited** option compares the language part of the regional code that is derived from the browser's display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will provide the regional code to websites as the default locale. If the OS-specified formatting language doesn't match Microsoft Edge's preferred language, the **Limited** option will act as if **Never** was selected (the regional code falls back to the browser's preferred display language). 
+
+Note: the **Limited** option is similar to current Firefox behavior for exposing the regional code, as described in [Regional Preferences - Firefox Source Docs documentation (mozilla.org)](https://firefox-source-docs.mozilla.org/intl/locale.html#regional-preferences), but also offers the possibility to not share this information at all. 
```

Comment:
```suggestion
Note: the **Limited** option is similar to current Firefox behavior for exposing the OS regional code, as described in [Regional Preferences - Firefox Source Docs documentation (mozilla.org)](https://firefox-source-docs.mozilla.org/intl/locale.html#regional-preferences).
```
I think we can drop the part where we brag about having more flexibility than Firefox (especially if it isn't 100% true).